MANIFEST.in

```diff
@@ -4,4 +4,5 @@ graft test
 prune docs/build
 prune test/http2-frame-test-case
 include README.rst LICENSE CHANGELOG.rst CONTRIBUTORS.rst tox.ini .gitmodules
+include src/hyperframe/py.typed
```

Comment:
Does this have the same effect as adding it to `package_data` in the setup call?

Reply:
I'll test this, but I think I should add to package_data as well